src/FilterWrapper/FilterWrapper.tsx

```diff
@@ -48,6 +53,25 @@ export class FilterWrapper extends React.Component<Props> {
 								{clearButtonCaption}
 							</button>
 						</InfoWrapper>
+						{selectionState !== "none" && (
```

Comment:
И использовать уже так
```suggestion
						{selectionState !== SelectionStateType.None && (
```